src/app/_utilities/formatList.ts

```diff
@@ -0,0 +1,13 @@
+export const formatList = (list: string[]): string => {
```

Comment:
a small suggestion for a function could be to use reduce here instead to keep it a little shorter:
```
const formatList = (list) => {
  if (list.length > 1) {
    list[list.length - 1] = `and ${list[list.length - 1]}`;
  }
  
  return list?.join(', ') || '';
}
```